R/implicit_integer_linter.R

```diff
@@ -2,6 +2,24 @@
 #'
 #' Check that integers are explicitly typed using the form `1L` instead of `1`.
 #'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "x <- 1",
```

Comment:
Given this comment, an example showing extraction would also be illustrative:

https://github.com/r-lib/lintr/issues/1155#issuecomment-1231709614

Reply:
Good idea.